src/main/java/com/nongfenqi/nexus/plugin/rundeck/RundeckMavenResource.java

```diff
@@ -102,7 +104,24 @@ public Response content(
             return commitAndReturn(NOT_FOUND, tx);
         }
 
-        String path = groupId.replace(".", "/") + "/" + artifactId + "/" + version + "/" + artifactId + "-" + version + ".jar";
+        if ("LATEST".equals(version)) {
+            List<RundeckXO> latestVersion = version(1, repositoryName, groupId, artifactId, classifier, extension);
+            if(latestVersion.isEmpty()){
+                return NOT_FOUND;
```

Comment:
`return commitAndReturn(NOT_FOUND, tx);`
because tx is open

Reply:
Thanks for the fix, I was seeing the exceptions, but wasn't sure how to fix it

Reply:
You can use 1.0.1,I fixed it.
And now the version param defalut is latest, so you incoming value 'LATEST' is invalid